.circleci/config.yml

```diff
@@ -171,6 +171,12 @@ jobs:
       - android/accept-licenses
       - android-dependencies
       - android/restore-build-cache
+      - run:
+          name: Verify purchases-android SDK compatibility
```

Comment:
wonder if we should also just put "SDK level 33" here so it's very obvious exactly what went wrong if this fails and you don't have context

Reply:
Hmm right... I'm concerned that we might forget about updating this in the future, when we need to update the target sdk... But maybe that's ok we can work it so it makes sense 👍